packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/debug/interfaces/DeveloperSettings.kt

```diff
@@ -7,35 +7,39 @@
 
 package com.facebook.react.modules.debug.interfaces
 
+import com.facebook.react.packagerconnection.PackagerConnectionSettings
+
 /** Provides access to React Native developers settings. */
 public interface DeveloperSettings {
+  /** Return the underlying [PackagerConnectionSettings] instance. */
+  public val packagerConnectionSettings: PackagerConnectionSettings
 
-  /** whether an overlay showing current FPS should be shown. */
-  public fun isFpsDebugEnabled(): Boolean
+  /** Whether an overlay showing current FPS should be shown. */
+  public var isFpsDebugEnabled: Boolean
```

Comment:
@Kudo This question applies to all `fun` -> `var` changes, but it appears as you are taking a read-only interface and making it read-write for everyone. Is this a safe change to make?
Also if the `internal` visibility `DevInternalSettings` is of an issue, would it be just better to make it `public`?